src/main/java/org/beanpod/switchboard/dto/mapper/CycleAvoidingMappingContext.java

```diff
@@ -0,0 +1,22 @@
+package org.beanpod.switchboard.dto.mapper;
+
+import org.mapstruct.BeforeMapping;
+import org.mapstruct.MappingTarget;
+import org.mapstruct.TargetType;
+
+import java.util.IdentityHashMap;
+import java.util.Map;
+
+public class CycleAvoidingMappingContext {
```

Comment:
That's a leftover of somethin I tried to avoid cycles in mapstrcut mappers. I took the code from [here](https://github.com/mapstruct/mapstruct-examples/tree/master/mapstruct-mapping-with-cycles). It's not being used now though so I removed it.